WooCommerce/Credentials/Templates/InfoPlist-Template.h

```diff
@@ -7,12 +7,12 @@
 
 /// Google Auth Callback Scheme
 ///
-#define GOOGLE_AUTH_SCHEME <#Google Auth Scheme#>
```

Comment:
We can't use the template syntax here since this get put into a plist file. the `<` causes it to fail to compile.